### ISSUE TITLE
Add PURLs for Apollo-SV modules

### ISSUE THREAD

**Issue by jamesaoverton** [OBOFoundry/purl.obolibrary.org#502]
Apollo-SV would like PURLs for several modules. @aellenhicks included the following list in an email to OBO Discuss today: 

> Title: Apollo-SV Region Neutral Instances and Classes
> Requested PURL: http://purl.obolibrary.org/obo/apollo_sv_region_neutral_instances_and_classes.owl
>
> Title: Apollo-SV Instances and Classes of the Continent of South America
> http://purl.obolibrary.org/obo/apollo_sv-instances-and-classes-continent-of-South-America.owl
>
> Title: Apollo-SV Instances and Classes of the Continent of North America
> http://purl.obolibrary.org/obo/apollo_sv-instances-and-classes-continent-of-North-America.owl
>
> Title: Apollo-SV Instances and Classes of the Continent of Europe
http://purl.obolibrary.org/obo/apollo_sv-instances-and-classes-continent-of-Europe.owl
>
>Title: Apollo-SV Instances and Classes of the Continent of Australia
> http://purl.obolibrary.org/obo/apollo_sv-instances-and-classes-continent-of-Australia.owl
>
> Title: Apollo-SV Instances and Classes of the Continent of Asia
> http://purl.obolibrary.org/obo/apollo_sv-instances-and-classes-continent-of-Asia.owl
>
> Title: Apollo-SV Instances and Classes of the Continent of Africa
> http://purl.obolibrary.org/obo/apollo_sv-instances-and-classes-continent-of-Africa.owl

Apollo-SV has a PURL configuration file here: https://github.com/OBOFoundry/purl.obolibrary.org/blob/master/config/apollo_sv.yml

The requested PURLs don't conform to our OBO ID policy <http://obofoundry.org/id-policy.html>, which requires a slash after the ontology ID, e.g. <http://purl.obolibrary.org/obo/apollo_sv/region_neutral_instances_and_classes.owl>. @aellenhicks Is this change acceptable?

**Comment by aellenhicks** [comment]
Hi James,

That's acceptable. Thank you. Apologies for the error.

**Comment by jamesaoverton** [comment]
Ok. I'll make a Pull Request.

Is there a reason why only one of these PURLs uses undescores, while the others use hyphens? It doesn't matter to me, but users appreciate consistency.

**Comment by aellenhicks** [comment]
Thanks. That was a typo. How is this?

Title: Apollo-SV Region Neutral Instances and Classes
Requested PURL: http://purl.obolibrary.org/obo/apollo_sv/region-neutral-instances-and-classes.owl

Title: Apollo-SV Instances and Classes of the Continent of South America
http://purl.obolibrary.org/obo/apollo_sv/instances-and-classes-continent-of-South-America.owl

Title: Apollo-SV Instances and Classes of the Continent of North America
http://purl.obolibrary.org/obo/apollo_sv/instances-and-classes-continent-of-North-America.owl

Title: Apollo-SV Instances and Classes of the Continent of Europe
http://purl.obolibrary.org/obo/apollo_sv/instances-and-classes-continent-of-Europe.owl

Title: Apollo-SV Instances and Classes of the Continent of Australia
http://purl.obolibrary.org/obo/apollo_sv/instances-and-classes-continent-of-Australia.owl

Title: Apollo-SV Instances and Classes of the Continent of Asia
http://purl.obolibrary.org/obo/apollo_sv/instances-and-classes-continent-of-Asia.owl

Title: Apollo-SV Instances and Classes of the Continent of Africa
http://purl.obolibrary.org/obo/apollo_sv/instances-and-classes-continent-of-Africa.owl

**Comment by jamesaoverton** [comment]
Sorry, I'm not thinking straight. Where are the OWL files that these PURLs should point to? I need the kind of information that's in this file: <https://github.com/OBOFoundry/purl.obolibrary.org/blob/master/config/apollo_sv.yml>. The PURL entries should end up looking like this:

```
- exact: /instances-and-classes-continent-of-Africa.owl
  replacement: https://raw.githubusercontent.com/ApolloDev/apollo-sv/master/src/ontology/instances-and-classes-continent-of-Africa.owl
```

The `replacement` field has to point to an OWL file. I was expecting to see something like `instances-and-classes-continent-of-Africa.owl` in the Apollo-SV GitHub repository <https://github.com/ApolloDev/apollo-sv>, but I don't.

**Comment by aellenhicks** [comment]
Fair point. We originally had these in an internal application ontology in a different repo. I'm coordinating with our team to now to ensure that moving them to the Apollo-SV doesn't break anything.

Apologies of the inconvenience.

**Comment by aellenhicks** [comment]
Thanks, James. I have updated the .yml file to include the desired redirects

**Comment by jamesaoverton** [comment]
Sorry for the delay. Can you please email me a copy of the updated file? (Or make a Pull Request here on GitHub, if you're familiar with that process.)
